### DISCUSSION TITLE
Integration of CVAT to Openstack swift

### DISCUSSION THREAD

**Discussion by keshva23** [cvat-ai/cvat#4091] [Q&A]
We wanted to use OpenStack(Private cloud) to store the files(Annotated images and videos) from the CVAT tool. 

We have explored OpenStack Swift . 

Wanted to know , how can we integrate the CVAT tool to upload the files to OpenStack.

Or Can you please suggest , how can we upload files from CVAT application to OpenStack.

**Comment by nmanovic** [comment]
@keshva23 , I'm afraid we don't have such experience. We can try to answer on specific questions.

**Reply by keshva23** [reply]
Ok. Thank you @nmanovic  .
